src/cmt_encode_prometheus_remote_write.c

```diff
@@ -70,11 +70,6 @@ static int pack_metric_metadata(struct cmt_prometheus_remote_write_context *cont
 static int append_metric_to_timeseries(struct cmt_prometheus_time_series *time_series,
                                        struct cmt_metric *metric);
 
-static int pack_metric_sample(struct cmt_prometheus_remote_write_context *context,
```

Comment:
This is ok, however, it needs to be replaced with the definition of `pack_basic_metric_sample` and pack_complex_metric_sample` but that can be done without haste.